health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/hooks/services/createUpdatePlanProject.js

```
@@ -883,33 +915,33 @@ const createUpdatePlanProject = async (req) => {
           Digit.Utils.microplanv1.updateUrlParams({ internalKey: null });
           return {
             triggeredFrom,
           };
-          }else{
-            setShowToast({ key: "error", label: "ERR_FAILED_TO_UPDATE_PLAN" });
-          }
+        } else {
+          setShowToast({ key: "error", label: "ERR_FAILED_TO_UPDATE_PLAN" });
         }
-        
+      }
```

Biome (1.9.4)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)